ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobMetadata.java

```diff
@@ -276,6 +277,23 @@ public CloudBlobMetadata setVcrKmsContext(String vcrKmsContext) {
     return this;
   }
 
+  /**
+   * @return the blob's name in cloud.
+   */
+  public String getCloudBlobName() {
+    return cloudBlobName;
+  }
+
+  /**
+   * Sets blob's name in cloud.
+   * @param cloudBlobName the blob's name in cloud.
+   * @return this instance.
+   */
+  public CloudBlobMetadata setCloudBlobName(String cloudBlobName) {
+    this.cloudBlobName = cloudBlobName;
+    return this;
```

Comment:
Minor and optional: I see multiple set methods in this class return `this`. I feel like we may use `CloudBlobMetadataBuilder` to instantiate `CloudBlobMetadata`. Currently, most set methods are never called.

Reply:
According to @lightningrob setMethod is used by cosmoDB client.

Let me think about builder.

Reply:
Will not make change this time.